composer.json

```diff
@@ -21,10 +21,11 @@
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
         "doctrine/phpcr-odm": "^1.4",
+        "jackalope/jackalope": "^1.0 || ^2.0@dev",
```

Comment:
Not sure why this is now required:

https://github.com/dantleech/phpcr-migrations-bundle/actions/runs/4436191849/jobs/7784305593

![Bildschirmfoto 2023-03-16 um 11 50 45](https://user-images.githubusercontent.com/1698337/225595104-353c0b24-e858-4508-8a0a-825e97087fc2.png)

Maybe @dbu has a hint

Reply:
Also `^2.0@dev` currently not possible because of:

https://github.com/dantleech/phpcr-migrations-bundle/pull/20#discussion_r1138464459

![Bildschirmfoto 2023-03-16 um 11 53 38](https://user-images.githubusercontent.com/1698337/225596123-d879514a-e068-41d7-bb6e-3edc1cad8323.png)

Reply:
looks like i need to upgrade the phpcr-bundle for jackalope 2 - i did not do that yet.

for the need to require jackalope: i think this has been necessary before as well - phpcr-odm does not require a specific phpcr implementation. is the ci build of this bundle not requiring jackalope dynamically?

Reply:
> for the need to require jackalope: i think this has been necessary before as well - phpcr-odm does not require a specific phpcr implementation. is the ci build of this bundle not requiring jackalope dynamically

It seems not: https://github.com/dantleech/phpcr-migrations-bundle/blob/master/.github/workflows/ci.yaml

There is no log of the last run but atleast last year it seems to have run without problems without require a phpcr implementation 🤔 :

https://github.com/dantleech/phpcr-migrations-bundle/commits/master